generators/k8helmpipeline/index.js

```diff
@@ -61,26 +54,21 @@ module.exports = class extends Generator {
          // of the generator
          this.pat = util.reconcileValue(cmdLnInput.options.pat, answers.pat);
          this.tfs = util.reconcileValue(cmdLnInput.options.tfs, answers.tfs);
-         this.type = util.reconcileValue(cmdLnInput.options.type, answers.type);
+         this.type = util.reconcileValue(cmdLnInput.options.type, ``,``);
```

Comment:
seems odd to me that this is an empty string versus answer.type

Reply:
I set `cmdLnInput.options.type = 'kubernetes';` before the user is prompted. The user is not prompted for a type because if they are running this sub-generator, they are deploying to Kubernetes